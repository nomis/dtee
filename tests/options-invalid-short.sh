#!/bin/bash
set -vx
. "$(dirname "$0")"/util/common.sh

run_test "--debug-options" "-a" "-y" "-b" "-a" "-b" "-t" "-u"
RET=$?

echo RET $RET
if [ $RET -eq 64 ]; then
	exit 0
fi
exit 1
