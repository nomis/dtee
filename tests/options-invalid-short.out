dtee: unrecognised option '-a'
Try 'dtee -h' for more information.
